src/pyvesync/vesync.py

```diff
@@ -72,22 +73,84 @@ def energy_update_interval(self, new_energy_update):
         if new_energy_update > 0:
             self._energy_update_interval = new_energy_update
 
+    @staticmethod
+    def device_search(device, new_list):
```

Comment:
`device_search` makes it sound more like this method would return whether a device exists as opposed to what the docstring says this method does. what do you think on renaming this to something more clearer?

Reply:
Agreed, renamed to `remove_dev_test`